pkg/handlers/system_intakes_test.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func newMockFetchSystemIntakes(systemIntakes models.SystemIntakes, err error) fetchSystemIntakes {
```

Comment:
I would add a handler test for what happens if a user passes a filter that doesn't exist. For example, if I query `system_intakes?status=BLUE`, does the filter fail silently? Does it throw a bad request error? I would probably lean toward a bad request error with a good error message, but open to whatever functionality you think is best. And then let's document that decision with a test.